### PR TITLE
Library updates: spring-security, spring-security-saml, spring-boot.

### PR DESCRIPTION
**Pull request by antonevane**
Updated spring-security-saml2-core to 1.0.4.RELEASE version.

### CODE REVIEW

**Comment by ulisesbocchio** [comment]
Thanks @antonevane! Would you take the look at the build from this PR? It's failing with some dependency problems

**Comment by antonevane** [comment]
@ulisesbocchio The build has been fixed.

File: spring-boot-security-saml/pom.xml [hunk 1]
```diff
@@ -35,7 +35,12 @@
         <dependency>
             <groupId>org.springframework.security.extensions</groupId>
             <artifactId>spring-security-saml2-core</artifactId>
-            <version>1.0.2.RELEASE</version>
+            <version>1.0.4.RELEASE</version>
+        </dependency>
+        <dependency>
```
Comment (ulisesbocchio) [inline spring-boot-security-saml/pom.xml]: why do we need this?

File: pom.xml [hunk 2]
```diff
@@ -41,7 +41,22 @@
...
+      </repository>
+      <repository>
+        <id>mulesoft</id>
+        <url>http://repository.mulesoft.org/releases/</url>
+      </repository>
+      <repository>
+        <id>alfresco</id>
+        <url>https://artifacts.alfresco.com/nexus/content/repositories/public/</url>
+      </repository>
+    </repositories>
```
Comment (ulisesbocchio) [inline pom.xml]: is this a set of repos you use?

File: spring-boot-security-saml/pom.xml (hunk 1 above)
Comment (antonevane) [reply spring-boot-security-saml/pom.xml]: This is the latest spring-security-saml library. Unless I don't undestand the question.

File: pom.xml (hunk 2 above)
Comment (antonevane) [reply pom.xml]: Yes. It is the set of repositories for spring saml.

File: spring-boot-security-saml/pom.xml (hunk 1 above)
Comment (ulisesbocchio) [reply spring-boot-security-saml/pom.xml]: I was talking about the json dependency

Comment (antonevane) [reply spring-boot-security-saml/pom.xml]: Ohh, The github showed me the saml dependency. OK. I was fixing the build and forgot that this dependency has been repacjed under the spring ubmrella. I'll remove it.

Comment (antonevane) [reply spring-boot-security-saml/pom.xml]: Done

**Comment by trblft** [comment]
How can I force the SAML2 core version to 1.0.4 in pom.xml please? I tried to use an exclusion but it fails.
```
              <dependency>
		    <groupId>com.github.ulisesbocchio</groupId>
		    <artifactId>spring-boot-security-saml</artifactId>
		    <version>1.13</version>
		    <exclusions>
		    <exclusion>
    			<groupId>org.springframework.security.extensions</groupId>
    			<artifactId>spring-security-saml2-core</artifactId>
		    </exclusion>
		    </exclusions>
		</dependency>
		<dependency>
    		<groupId>org.springframework.security.extensions</groupId>
    		<artifactId>spring-security-saml2-core</artifactId>
    		<version>1.0.4.RELEASE</version>
		</dependency>
```

**Comment by trblft** [comment]
Ok, this update solved almost all my problems, I'd be glad to find this on Maven central, please, when have time, have a nice day guys.

**Comment by ulisesbocchio** [comment]
I will merge and release his ASAP

**Comment by trblft** [comment]
Thanks a lot to both of you, great timesaving piece of software!

**Pull request merged**